x-pack/test/functional/apps/graph/graph.ts

```diff
@@ -47,67 +46,57 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
       '/blog/wp-admin/',
     ];
 
-    const expectedConnectionWidth: Record<string, Record<string, number>> = {
-      '/blog/wp-admin/': { wp: 2, blog: 5.51581 },
+    const expectedConnections: Record<string, Record<string, boolean>> = {
```

Comment:
After syncing with @markharwood it doesn't make sense to validate specific line lengths because the exact numbers can change because of a lot of things.